content/docs/reference/api/aut/op-prot/_index.md

```diff
@@ -830,7 +830,11 @@ aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D setSymbol
 
 {{< tabpane langEqualsHeader=true >}}
 {{< tab header="aut" >}}
+<<<<<<< HEAD
 aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D setSymbols ["AUD/USD", "CAD/USD", "EUR/USD", "GBP/USD", "JPY/USD", "SEK/USD", "ATN/USD", "NTN/ATN"]
+=======
+aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D setSymbols ["AUD-USD", "CAD-USD", "EUR-USD", "GBP-USD", "JPY-USD", "SEK-USD", "ATN-USD", "NTN-USD"]
+>>>>>>> b68833b (Oracle, edits to add Docker run and install)
```

Comment:
more merge conflicts

Reply:
Sorry 😞 . Fixed in commit 480540aa065b97910aa107113b7c3d671bcbd22c